### PR TITLE
ci: Delete the previous kata image

### LINKED ISSUE #3796: Delete the previous kata image
**Which feature do you think can be improved?**

Make it possible to run the CI a few times without rebooting the machine.
**How can it be improved?**

The scripts sometime fail because the kata image is not deleted.
Improve the deletion related code.

### PR DESCRIPTION
**Pull request by marcel-apf**
Improve the code removing the kata image from the previous run.

fixes #3796

Signed-off-by: Marcel Apfelbaum <marcel@redhat.com>

### CODE REVIEW

File: .ci/install_kata_image.sh [hunk 1]
```diff
@@ -58,9 +58,9 @@ build_rust_image() {
 			;;
 	esac
 	info "Install ${target_image} to ${image_path}"
-	if [ -f "${image_path}" ]; then
+	local file="$(realpath ${image_path}/$(basename ${file_to_install}))"
+	if [ -f "${image_path}"  ] || [ -f "${file}" ]; then
```
Comment (devimc) [inline .ci/install_kata_image.sh:62]: thanks - I think that `if [ -f "${file}" ]; then` should be enough

Comment (devimc) [reply .ci/install_kata_image.sh:62]: @marcel-apf `$image_path` is a directory, we should ignore it, right? wdyt?

Comment (marcel-apf) [reply .ci/install_kata_image.sh:62]: @devimc sure, I'll respin soon(ish)

Comment (marcel-apf) [reply .ci/install_kata_image.sh]: @devimc done

**Review by devimc — APPROVED** [review]
thanks @marcel-apf

**Review by snir911 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**